ppr-ui/src/composables/mhrRegistration/useNewMhrRegistration.ts

```diff
@@ -138,6 +138,13 @@ export const useNewMhrRegistration = () => {
     setMhrHomeDescription({ key: 'manufacturer', value: data.description.manufacturer })
     setMhrHomeDescription({ key: 'certificationOption', value: HomeCertificationOptions.CSA })
     setMhrRegistrationSubmittingParty(data.submittingParty)
+
+    // Add id to owners to ensure home owners table functionality works.
+    let defaultID = 1
+    data.ownerGroups.forEach((group: MhrRegistrationHomeOwnerGroupIF) => {
```

Comment:
Is it required only here? Otherwise could be added to the getter itself.

Reply:
Good question I am not sure. Maybe @doug-lovett has some insights into this.

This is the first case where I have seen us fetching owners without ownerIds from database.

According to the model it is an optional field, but some of the current functionality does seem to rely on it.
For example if there is no ID, it does not render the owners in the homeOwnersTable.

From discussions with cameron, it does seem to be a superficial field (used for UI purposes), unlike groupID 

Something that we should consider, but I think its outside the scope of this ticket.

Reply:
We do use IDs just for rendering UI components. If this is the case, we can keep it here, I'm ok with that.